<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RedisInfoDialog</class>
 <widget class="QDialog" name="RedisInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="_hostLabel">
         <property name="text">
          <string>主机:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="_hostComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="_itemLabel">
         <property name="text">
          <string> 选项:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="_itemComboBox"/>
       </item>
       <item>
        <spacer name="_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="_queryPushButton">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_exitPushButton">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="_textBrowser"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
